src/app/front-page/dialog/dialog.component.ts

```diff
@@ -16,7 +16,9 @@ export class DialogComponent implements OnInit {
   ngOnInit() {
   }
 
+
   login() {
+    //@khaled TODO : activate google tracking without cookies , in case the user rejected the cookies.
```

Comment:
@Kalud-Z still a todo?

Reply:
no, not really. I will delete it, and remove also all unnecessary comments.